DESCRIPTION

```diff
@@ -32,10 +32,13 @@ Imports:
     desc,
     digest,
     downlit,
+    evaluate,
```

Comment:
Would you mind putting these packages in suggests rather than imports? They do belong in imports, but we are starting to hit the CRAN limit of 20ish imported packages, and it's easier to just work around it than argue about it. I don't think this change would have any material impact because the three packages are either already indirect hard dependencies (i.e. downlit imports evaluate) or the code that uses them won't get called unless the package is installed (i.e. there's no way to get HTML dependencies until htmltools is installed).

Reply:
Sure, I'll see how much of that I can do.

Reply:
All done now.